### ISSUE TITLE
Add SMTP host configs to the docs

### ISSUE THREAD

**Issue by Methma** [wso2/docs-apim#1146]
**Description:**
Please add a note about other system properties of email sink to the doc [1], 1st point under 'Subscribing to emails based on Publisher alerts' as many customers are using SMTP hosts other than Gmail. 

Please find the sample configurations below. Also for further information please refer [2].

```
- extension:
--
  | name: email
  | namespace: sink
  | properties:
  | username: <senders-email-username>
  | address: <senders-email-address>
  | password: <senders-email-password>
  | mail.smtp.host: <senders-email-host>
  | port: 25
  | ssl.enable : false
  | auth : false
  | mail.store.protocol: pop3
```

[1]. https://docs.wso2.com/display/AM260/Configuring+Alerts
[2]. https://siddhi-io.github.io/siddhi-io-email/api/latest/

**Suggested Labels:**
apim-analytics

**Affected Product Version:**
apim-analytics-2.6.0 , apim-analytics-3.0.0, apim-analytics-3.1.0

**Comment by Mariangela** [comment]
@fazlan-nazeem , need some clarification on this.

**Comment by ruwiniwj** [comment]
@Mariangela i think it will be sufficient to add a doc link to the siddhi docs from APIM docs than to duplicate the content in APIM docs. Eg: We can have a mention under  'Subscribing to emails based on Publisher alerts'[1] saying 'For further information on email configuration please refer ...' and point to [2]. 

**Note**: We need to point the docs of corresponding siddhi version used in analytics. 

[1]. https://docs.wso2.com/display/AM260/Configuring+Alerts#ConfiguringAlerts-SubscribingtoemailsbasedonPublisheralerts
[2]. https://siddhi-io.github.io/siddhi-io-email/api/latest/

**Comment by ruwiniwj** [comment]
@Mariangela i reassessed the requirement and the docs and i think it would be clearer to add an info box under step 1 sub-step b(i.e. '_Configure the sender email address. The sample code is shown below._')of [1] to indicate that if we are configuring an SMTP host other gmail the email sender configuration should be as follows

```
siddhi:
  extensions:
...
    -
      extension:
        name: email
        namespace: sink
        properties:
          username: <senders-email-username>
          address: <senders-email-address>
          password: <senders-email-password>
          mail.smtp.host: <senders-email-host>
          port: 25
          ssl.enable : false
          auth : false
          mail.store.protocol: pop3
...
```

[1]. https://docs.wso2.com/display/AM260/Configuring+Alerts#ConfiguringAlerts-SubscribingtoemailsbasedonPublisheralerts

**Comment by Mariangela** [comment]
Fixed in API-M 2.6.0 docs
https://docs.wso2.com/display/AM260/Configuring+Alerts

**Comment by Mariangela** [comment]
Fixed in API-M 3.0.0 docs.
https://github.com/wso2/docs-apim/pull/2289

**Comment by Mariangela** [comment]
Fixed in API-M 3.1.0 docs.
https://github.com/wso2/docs-apim/pull/2293

**Comment by Mariangela** [comment]
Fixed in APIM 3.2.0 docs
https://github.com/wso2/docs-apim/pull/2294

**Comment by Mariangela** [comment]
Fixed in the APIM master branch docs
https://github.com/wso2/docs-apim/pull/2296
